<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioIODialog</class>
 <widget class="QDialog" name="AudioIODialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio I/O</string>
  </property>
  <widget class="QLabel" name="asioDriverLabel">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>31</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ASIO driver</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboAsioDriver">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>31</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sampleRateLabel">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>261</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="bufferSizeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Buffer size</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboSampleRate">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>261</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBufferSize">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>300</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxInputs">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>67</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>281</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="firstInputLabel">
       <property name="text">
        <string>First</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboFirstInput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastInputLabel">
       <property name="text">
        <string>Last</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboLastInput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxOutputs">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Outpus</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>281</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="firstOutputLabel">
       <property name="text">
        <string>First</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboFirstOutput"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastOutputLabel">
       <property name="text">
        <string>Last</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboLastOutput"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>released()</signal>
   <receiver>AudioIODialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
